src/services/mongo.ts

```diff
@@ -1,24 +1,10 @@
-import {
-  mongoUsernameKey,
-  mongoUserpassKey,
-  vaultMongoCredentialPath,
-  mongoUser,
-  mongoPass,
-  mongoHost,
-  mongoDb,
-} from '../config';
-
-import vault from './vault';
 import mongoose from './mongoose';
+import getCredentials from './vault';
 
-export const getMongoCredentials = async () => {
-  return vaultMongoCredentialPath
-    ? await vault.getSecretValue(vaultMongoCredentialPath).then(r => ({
-        user: r[mongoUsernameKey],
-        pass: r[mongoUserpassKey],
-      }))
-    : { user: mongoUser, pass: mongoPass };
-};
+import { mongoHost, mongoDb, mongoUser, mongoPass } from '../config';
+
+export const getMongoCredentials = async () =>
+  (await getCredentials()) || { user: mongoUser, pass: mongoPass };
```

Comment:
with `USE_VAULT` now I'm thinking we might wanna just throw an error and exit the process rather than falling back on env here